src/BenchmarkDotNet/Running/BenchmarkRunnerDirty.cs

```diff
@@ -16,18 +20,22 @@ namespace BenchmarkDotNet.Running
     // so it can ignore the version mismatch ;)
     public static class BenchmarkRunner
     {
+        private static readonly IUserInteraction userInteraction = new UserInteraction();
+        private static readonly List<Type> types = new List<Type>();
+        private static readonly List<Assembly> assemblies = new List<Assembly>();
```

Comment:
this field is never used for adding new assemblies, please remove it and just pass an empty list to `TypeFilter.GetTypesWithRunnableBenchmarks`